utility_code/defragment.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env pypy3
+
+import os
+import sys
+import getopt
+import multiprocessing
+import concurrent
+import traceback
+from minecraft.world import World
+from lib_py3.common import eprint
+from lib_py3.timing import Timings
+
+def usage():
+    """Prints usage and exits with error"""
+    sys.exit(f"Usage: {sys.argv[0]} [--verbose] [--num-threads 4] /path/to/world [/path/to/parent ...]")
+
+def disk_usage_for_path(path):
+    """Gets approximate real disk usage for a path"""
+    # TODO: Everything I've tried is garabage, this is definitely wrong
+    # Needs replacing with a function that actually works
+    return os.stat(path).st_blocks * 512
```

Comment:
I left some suggestions for this in server ops, hopefully one of them is helpful.